extensions/amp-sidebar/1.0/component.js

```diff
@@ -238,14 +292,16 @@ export function SidebarToolbar({
   }, [mediaQuery, toolbarTarget, targetEl]);
 
   return (
-    <nav
-      ref={ref}
-      toolbar={mediaQueryProp}
-      toolbar-target={toolbarTargetProp}
-      {...rest}
-    >
-      {children}
-    </nav>
+    preactMode && (
+      <nav
+        ref={ref}
+        toolbar={mediaQueryProp}
+        toolbar-target={toolbarTargetProp}
```

Comment:
I know this was here before but wanted to ask - Do these attributes need to be rendered on the nav element? Can they be removed?

Reply:
They can be removed.  Slight preference for keeping though to match previous functionality and so user can check the output against their input.